client/components/Item/Card/Card.vue

```diff
@@ -202,6 +203,39 @@ export default Vue.extend({
 
       return '';
     },
+    /**
+     * Gets a link to be applied to the card title
+     *
+     * @returns {string} A router link to the item or a related item
+     */
+    cardTitleLink(): string {
+      if (this.item.Type === 'Episode' && this.item.SeriesId) {
+        return this.getItemDetailsLink({ Id: this.item.SeriesId }, 'Series');
+      }
+
+      return this.getItemDetailsLink(this.item);
+    },
+    /**
+     * Gets a link to be applied to the card subtitle
+     *
+     * @returns {string|undefined} A router link to the parent item or a related item
+     */
+    cardSubtitleLink(): string | undefined {
+      if (
+        this.item.Type === 'MusicAlbum' &&
+        this.item.AlbumArtists &&
+        this.item.AlbumArtists.length > 0
+      ) {
+        return this.getItemDetailsLink(
+          this.item.AlbumArtists[0],
+          'MusicArtist'
+        );
+      } else if (this.item.Type === 'Episode') {
+        return this.getItemDetailsLink(this.item);
+      }
+
+      return undefined;
```

Comment:
This is unnecessary. Or return ``null`` if we explicitly want to know that this returned something.

Reply:
The return is necessary here cause it's an if else-if condition, and ESLint throws if it itsn't here

Reply:
Returns null now